python/src/GaussianLinearCalibration_doc.i.in

```diff
@@ -48,6 +48,9 @@ The resulting error distribution is :class:`~openturns.Normal`
 with a zero mean and with a covariance matrix equal to the
 `errorCovariance` input argument.
 
+Please read read :ref:`gaussian_calibration` for more details
+on Gaussian calibration.
```

Comment:
```suggestion
Please read :ref:`gaussian_calibration` for more details.
```
The title is already displayed, which makes this sentence look redundant in the HTML:
![image](https://user-images.githubusercontent.com/31989332/231948165-a745ea20-3723-4439-aac2-76951c6d255e.png)